tests/test_initialization.py

```diff
@@ -329,6 +331,374 @@ def test_lora_pissa_linear_init_default(self, data):
         peft_model = get_peft_model(deepcopy(model), config)
         assert torch.allclose(output, peft_model(data)[0], atol=1e-06)
 
+    def test_lora_corda_linear_init_default(self, data, tmp_path):
```

Comment:
Sure, I've extracted all CorDA tests to `class TestCordaInitialization`, and parametrized all tests over two CorDA methods. Note that when data is not enough, KPM fails to build an invertible covariance, causing KPM not to change base weights at all, so I've changed the `data` fixture so that it contains `1000` samples instead of `10` in `class TestCordaInitialization`.